test/browser/createContext.test.js

```diff
@@ -57,6 +57,43 @@ describe('createContext', () => {
 		expect(scratch.innerHTML).to.equal('<div><div>a</div></div>');
 	});
 
+	// This optimization helps
+	// to prevent a Provider from rerendering the children, this means
+	// we only propagate to children.
+	// Strict equal vnode optimization
+	it('should support wrappers', () => {
+		const { Provider, Consumer } = createContext();
+		let set,
+			renders = 0;
+		const Layout = ({ children }) => {
```

Comment:
The `Layout` is only a wrapper and for `Provider` this will never change since it's an external child so this will stop a lot of needless renders